webpack.config.js

```diff
@@ -132,15 +132,58 @@ function getWebpackConfig( env, argv ) {
 	const styleSheetFiles = /\.(sc|sa|c)ss$/i;
 	const scriptFiles = /\.[jt]sx?$/i;
 
-	webpackConfig.module.rules[ 3 ].generator.publicPath = '../';
+	const isProduction = process.env.NODE_ENV === 'production';
 
-	// Handle SVG images only in CSS files.
-	webpackConfig.module.rules[ 3 ].test = /\.(?:gif|jpg|jpeg|png|woff|woff2|eot|ttf|otf|svg)$/i;
-	webpackConfig.module.rules[ 3 ].issuer = styleSheetFiles;
+	webpackConfig.module.rules = webpackConfig.module.rules.map( ( rule ) => {
```

Comment:
Is it a rebase problem that these changes are here?

Reply:
Hey, it's because I decided to merge the feature branch back into this branch so I could generate a branch based on it, and the branch for sensei guides doesn't have the feature branch merged, so..yeah, kinda of. :| 

But it shouldn't generate any problems, I think, because I'm using merge commits instead of rebase :thinking: